src/modules/evaluateExpression/evaluateExpressionTestData.ts

```diff
@@ -691,11 +703,11 @@ testData.complex1 = {
             { value: 'SELECT COUNT(*) FROM user_organisation WHERE user_id = $1' },
             {
               operator: 'objectProperties',
-              children: [{ value: { object: 'user', property: 'id' } }],
+              children: [{ value: { objectIndex: 0, property: 'id' } }],
             },
             {
               operator: 'objectProperties',
-              children: [{ value: { object: 'organisation', property: 'id' } }],
+              children: [{ value: { objectIndex: 1, property: 'id' } }],
             },
```

Comment:
It is a good idea to uncomment the text for this `complex1` QuerySyntax to test this data changed

Reply:
Good call. The result of `complex1` is currently `false`, since there is no organisations in the database, but we should add some and make this test return `true` sometime soon.